Src/Generic/Styling/TextStyle.swift

```diff
@@ -60,7 +60,12 @@ extension TextStyle {
         style.font = HedvigFonts.circularStdBook!.withSize(17)
         style.color = .gray
     }
-
+    
+    static let rowTitlePurple = TextStyle.default.restyled { (style: inout TextStyle) in
```

Comment:
I think I've created a textstyle that looks exactly like this already 🙈

Reply:
`rowValueEditable` or maybe that that doesn't make sense

Reply:
Just changed the name to `rowValueLink`! 👍